### PR TITLE
Adding drop shadow offset-y and blur-radius menu option

### PR DESCRIPTION
**Pull request by rafaelcamaram**
Adding the option that allows the user to change the `box-shadow` _offset-y_ and _blur-radius_ properties.

### CODE REVIEW

File: components/Carbon.js [hunk 1]
```diff
@@ -107,7 +109,9 @@ class Carbon extends React.Component {
         <div id="container">
           {config.windowControls ? <WindowControls theme={config.windowTheme} /> : null}
           <CodeMirror
-            className={`CodeMirror__container window-theme__${config.windowTheme}`}
+            className={`CodeMirror__container CodeMirror__shadow window-theme__${
```
Comment (jakedex) [inline components/Carbon.js]: I don't think we need to add a new class (`CodeMirror__shadow`) here?

File: components/Carbon.js [hunk 2]
```diff
@@ -182,7 +186,16 @@ class Carbon extends React.Component {
               position: relative;
               z-index: 1;
               border-radius: 5px;
-              ${config.dropShadow ? 'box-shadow: 0 20px 68px rgba(0, 0, 0, 0.55);' : ''};
+            }
+
+            #container :global(.CodeMirror__shadow) {
+              ${config.dropShadow
```
Comment (jakedex) [inline components/Carbon.js]: Let's use template literals here

File: components/Settings.js [hunk 3]
```diff
@@ -44,6 +44,16 @@ class Settings extends React.Component {
             enabled={this.props.enabled.dropShadow}
             onChange={this.props.onChange.bind(null, 'dropShadow')}
           />
+          <Slider
+            label="Drop shadow offset-y"
```
Comment (jakedex) [inline components/Settings.js]: I think we should change these to 'Drop shadow (offset-y)' to be consistent with Padding (verticle), etc.

**Comment by mfix22** [comment]
Hey @rafaelcamaram, could you address the concerns @jakedex mentioned?

**Comment by rafaelcamaram** [comment]
@jakedex @mfix22 Sure! I'll fix it this week and commit it as soon as possible :)

**Comment by rafaelcamaram** [comment]
@jakedex @mfix22 I just did the changes that you mentioned :) If you think that something could be improved, please let me know :)

File: components/Settings.js [hunk 4]
```diff
@@ -44,6 +44,16 @@ class Settings extends React.Component {
             enabled={this.props.enabled.dropShadow}
             onChange={this.props.onChange.bind(null, 'dropShadow')}
           />
+          <Slider
+            label="Drop shadow (offset-y)"
+            value={this.props.enabled.dropShadowOffsetY || 20}
+            onChange={this.props.onChange.bind(null, 'dropShadowOffsetY')}
+          />
+          <Slider
+            label="Drop shadow blur-radius"
```
Comment (mfix22) [inline components/Settings.js]: Please make this one `"Drop shadow (blur-radius)"`

**Comment by mfix22** [comment]
Thanks @rafaelcamaram, just one more thing!

**Comment by rafaelcamaram** [comment]
@mfix22 Done! :)

**Comment by mfix22** [comment]
Thanks so much for the help @rafaelcamaram. Just released with your changes https://github.com/dawnlabs/carbon/releases/tag/1.0.4

**Pull request merged**